test/data/samples/fwjr_production.py

```diff
@@ -92,8 +92,8 @@
               'status': 0,
               'stop': 1454569736}
             ],
-'fallbackFiles': ['blahblah'],
-'skippedFiles': ['blahblah'],
+'fallbackFiles': [4],
```

Comment:
I think I understand now the usage of "blohblah" in fileArrays, it refers to fallbackFiles, right? If so can we modify the string to make it more clear, e.g. /lfn/fallbackfiles.root